ui/src/app/utils/index.tsx

```diff
@@ -42,16 +41,16 @@ export function isBlank(toTest: String): boolean {
   }
 }
 
+export const REGISTERED_TIER_SHORT_NAME = 'registered';
 /**
  * Determine whether the given access level is >= registered. This is the
  * minimum required level to do most things in the Workbench app (outside of
  * local/test development).
+ *
+ * TODO: make separate evaluations by tier
  */
-export function hasRegisteredAccess(access: DataAccessLevel): boolean {
-  return [
-    DataAccessLevel.Registered,
-    DataAccessLevel.Protected
-  ].includes(access);
+export function hasRegisteredAccess(accessTierShortNames: Array<string>): boolean {
+  return !!accessTierShortNames && accessTierShortNames.includes(REGISTERED_TIER_SHORT_NAME);
```

Comment:
I don't think there's a way for `accessTierShortNames` to be null/undefined here but I'm not 100% confident

Reply:
You could do:

```javascript
fp.includes(REGISTERED_TIER_SHORT_NAMES, accessTierShortNames)
```

It will return false if accessTierShortNames are `undefined` or `null` and will handle the case if a non array or string gets passed in. It will also prevent an error from being thrown if an object or number were to sneak in.